contracts/TreasuryManager.sol

```diff
@@ -149,6 +188,11 @@ contract TreasuryManager is
         emit NOTEPurchaseLimitUpdated(purchaseLimit);
     }
 
+    function setBALPurchaseLimit(uint256 purchaseLimit) external onlyOwner {
+        balPurchaseLimit = purchaseLimit;
```

Comment:
We should mark this as denominated in WETH.

Reply:
one